src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
```

Comment:
- [Optional] Kindly style your todo list to match the template in this [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) as demanded by project requirements.